crates/telio-nurse/src/heartbeat.rs

```diff
@@ -508,8 +491,21 @@ impl Analytics {
         }
     }
 
-    async fn handle_derp_event(&mut self, event: Server) {
-        self.derp_connection = event.conn_state == RelayState::Connected;
+    async fn handle_derp_event(&mut self, derp_event: Server) {
```

Comment:
Name is a bit misleading here. Earlier it was setting whether derp is connected or not, and now it is setting the current nat type.